src/main/java/com/intel/gkl/pairhmm/IntelPairHmm.java

```diff
@@ -68,10 +67,33 @@ public void initialize(PairHMMNativeArguments args) {
             args.useDoublePrecision = false;
             args.maxNumberOfThreads = 1;
         }
+
         if (args.useDoublePrecision && useFpga) {
             logger.warn("FPGA PairHMM does not support double precision floating-point. Using AVX PairHMM");
         }
+
+        if(!gklUtils.getFlushToZero()) {
+            logger.warn("Flush-to-zero (FTZ) is disabled; enabling just for PairHMM");
+        }
```

Comment:
Yes, we should not disable FTZ in the `done()` function. Once it is on, we will leave it on, which matches the current implementation.